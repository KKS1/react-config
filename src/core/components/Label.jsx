import React, { Component } from "react";

const Label = ({ schema, label, id }) => {
  return !schema && label ? <label htmlFor={id}>{label}</label> : "";
};

export default Label;
