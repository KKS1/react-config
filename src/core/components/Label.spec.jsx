import React, { Component } from "react";
import Label from "./Label";

describe("Label", () => {
  test("render_Label", () => {
    const props = {
      label: "loan question",
      id: "formOne_labaleOne"
    };
    const wrapper = shallow(<Label {...props} />);
    const instance = wrapper.getElement();
    expect(instance).toMatchSnapshot();
  });
});
