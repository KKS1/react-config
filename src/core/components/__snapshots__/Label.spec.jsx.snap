// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label render_Label 1`] = `
<label
  htmlFor="formOne_labaleOne"
>
  loan question
</label>
`;
